usecases/schema/class.go

```diff
@@ -99,13 +99,17 @@ func (h *Handler) GetCachedClass(ctxWithClassCache context.Context,
 func (h *Handler) AddClass(ctx context.Context, principal *models.Principal,
 	cls *models.Class,
 ) (*models.Class, uint64, error) {
-	err := h.Authorizer.Authorize(principal, authorization.CREATE, authorization.CollectionsMetadata()...)
+	cls.Class = schema.UppercaseClassName(cls.Class)
+	cls.Properties = schema.LowercaseAllPropertyNames(cls.Properties)
+
+	err := h.Authorizer.Authorize(principal, authorization.CREATE, authorization.CollectionsMetadata(cls.Class)...)
 	if err != nil {
 		return nil, 0, err
 	}
+	if err := h.Authorizer.Authorize(principal, authorization.READ, authorization.CollectionsMetadata(cls.Class)...); err != nil {
+		return nil, 0, err
+	}
```

Comment:
**suggestion:**  given that the entity is the same and the verb only , we can introduce verb 
in authorization package and it will check both at once 
```
CR = "(C)|(R)" 
```


for the delete class can be 
```
DR = "(D)|(R)" 
```